docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -793,11 +793,14 @@ If any of the two arguments is `null` or the pattern is an empty string `null` i
 If the 1st argument is of type `time`, then pattern specified by the 2nd argument cannot contain date related units
 (e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned.
 
-[NOTE]
-Format specifier `F` will be working similar to format specifier `f`.
+*Special Cases*
+
+- Format specifier `F` will be working similar to format specifier `f`.
 It will return the fractional part of seconds, and the number of digits will be same as of the number of `Fs` provided as input (up to 9 digits).
 Result will contain `0` appended in the end to match with number of `F` provided.
 e.g.: for a time part `10:20:30.1234` and pattern `HH:mm:ss.FFFFFF`, the output string of the function would be: `10:20:30.123400`.
+- Format Specifier `y` will be working similar to `yyyy`. eg.: For year `2009`, `y` will be returning `2009` instead of `9`.
```

Comment:
This is not 100% correct. if you have a year with 2 digits (let's say year 45) and use 3 or 4 `y` -> `yyy` or `yyyy` then you will get it with left padded 0s -> `045` and `0045`. Could you please add some test like that, and properly rephrase the docs?

Reply:
updated documentation and added test.